test/test_linear_physical_object.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+from src.linear_physical_object import LinearPhysicalObject
+from src.dimensions2d import Dimensions2D
+from src.vec2d import Vec2d
+
+
+class TestLinearPhysicalObject:
+    @pytest.fixture(autouse=True)
+    def setup(self):
+        self.dimensions = Dimensions2D(4, 8)
+        self.position = Vec2d(10, 50)
+        self.velocity = Vec2d(5, 50)
+        self.acceleration = Vec2d(-10, 2)
+        self.mass = 1e5
+
+    def test_init(self):
```

Comment:
In diesem Test keine Assertions?

Reply:
zu sehen, dass kein fehler passiert war mir erstmal genug